sdk/src/dlob/DLOB.ts

```diff
@@ -964,6 +1047,8 @@ export class DLOB {
 			throw new Error('Must provide OraclePriceData to get spot bids');
 		}
 
+		this.updateRestingLimitOrders(slot);
```

Comment:
looks like your missing calls to `updateRestingLimitOrders` in the other generator functions

Reply:
i think it only needs to be in the limit order generators, all the other generators just consume the limit order generators?